components/gateway/pkg/proxy/transport.go

```diff
@@ -68,22 +111,14 @@ func (t *Transport) RoundTrip(req *http.Request) (resp *http.Response, err error
 	resp.Body = ioutil.NopCloser(bytes.NewReader(responseBody))
 	defer httpcommon.CloseBody(resp.Body)
 
-	isMutation, err := checkQueryType(requestBody, "mutation")
-	if err != emptyQuery {
-		if err != nil {
-			return nil, err
-		}
-		if !isMutation {
-			log.Println("Will not auditlog queries")
-			return resp, nil
-		}
-	}
-
-	correlationHeaders := correlation.HeadersForRequest(req)
-	correlationCtx := context.WithValue(context.Background(), correlation.HeadersContextKey, correlationHeaders)
-	err = t.auditlogSvc.Log(correlationCtx, string(requestBody), string(responseBody), claims)
+	err = t.auditlogSink.Log(context.TODO(), AuditlogMessage{
```

Comment:
Use `req.Context()` here, no matter that it is not used at the moment by the Sink implementation. It might be used in future when logging in enhanced